mantl-cadvisor/mantl-cadvisor/spec.yml

```diff
@@ -0,0 +1,33 @@
+name: mantl-cadvisor
```

Comment:
there doesn't appear to by anything in here that's mantl-specific, so should the name be "cadvisor"?